pylint/checkers/utils.py

```diff
@@ -535,6 +535,67 @@ def next_char(i):
     return keys, num_args, key_types, pos_types
 
 
+def parse_brace_format_string(format_string: str) -> Tuple[Set[str], int]:
```

Comment:
It's not totally obvious but we have `parse_format_method_string` in `checkers/strings.py`, which does the same thing. It's used by the string checkers for warning about passing invalid format keys when formatting with `.format()`.

Reply:
Thanks for pointing that out. I will refactor accordingly.